src/main/java/org/opentripplanner/middleware/models/ApiUser.java

```diff
@@ -44,19 +44,26 @@
      */
     @Override
     public boolean delete() {
+        return delete(true);
+    }
+
+    public boolean delete(boolean deleteAuth0User) {
```

Comment:
javadoc could use an update (this method probably deserves the main javadoc and the one above should describe it as a default shorthand that deletes the Auth0 user).

Reply:
Updated.